prediction-polls/backend/src/repositories/ProfileDB.js

```diff
@@ -0,0 +1,136 @@
+const authDb = require("../repositories/AuthorizationDB.js");
+const mysql = require('mysql2')
+
+require('dotenv').config();
+
+const pool = mysql.createPool({
+    host: process.env.MYSQL_HOST,
+    user: process.env.MYSQL_USER,
+    password: process.env.MYSQL_PASSWORD,
+    database: process.env.MYSQL_DATABASE
+}).promise()
+
+
+async function getProfileWithProfileId(profileId){
+    console.log("profileId")
+    console.log(profileId)
+
+    const sql = 'SELECT * FROM profiles WHERE id= ?';
+
+    try {
+        const [rows] = await pool.query(sql, [profileId]);
+        console.log(rows)
+        return rows;
+    } catch (error) {
+        console.error('getProfileWithProfileId(): Database Error');
+        throw error;
+    }
+}
+
+async function getProfileWithUserId(userId){
+    const sql = 'SELECT * FROM profiles WHERE user_id= ?';
+
+    try {
+        const [rows, fields] = await pool.query(sql, [userId]);
+        return {profile:rows[0]};
+    } catch (error) {
+        return {error:{code:1000,message:"An error occured:" + error}};
```

Comment:
For example, here you should pull this code and message from the errorCodes.js file